src/main/java/gameframework/game/GameUniverse.java

```diff
@@ -13,6 +14,8 @@
 
 	public void removeGameEntity(GameEntity gameEntity);
 
+	public ConcurrentLinkedQueue<GameEntity> getGameEntities();
```

Comment:
`ConcurrentLinkedQueue` is an implementation detail. Please use something much more abstract.